addons/sale_management/views/res_config_settings_views.xml

```diff
@@ -5,6 +5,7 @@
         <field name="name">res.config.settings.view.form.inherit.sale.management</field>
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="sale.res_config_settings_view_form"/>
+        <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'), 0)]"/>
```

Comment:
I would add it at div level in stable, as people may inherit from this view and may see it disappear after upgrading.

Reply:
Same as above.

I don't think it will disappear because, if they're not getting the error, it's because they have permissions. And in that case it will stay.